addons/theme_bootswatch/views/theme_bootswatch_templates.xml

```diff
@@ -6,21 +6,22 @@
-                    <opt data-xmlid="" data-icon="/theme_bootswatch/static/src/img/bootswatch_default_thumbnail.png"/>
-                    <opt data-xmlid="theme_bootswatch.theme_cerulean_bs_variables,theme_bootswatch.theme_cerulean" data-icon="/theme_bootswatch/static/src/img/cerulean_thumbnail.png"/>
-                    <opt data-xmlid="theme_bootswatch.theme_cosmo_bs_variables,theme_bootswatch.theme_cosmo" data-disable="ee" data-icon="/theme_bootswatch/static/src/img/cosmo_thumbnail.png"/>
-                    <opt data-xmlid="theme_bootswatch.theme_cyborg_bs_variables,theme_bootswatch.theme_cyborg" data-icon="/theme_bootswatch/static/src/img/cyborg_thumbnail.png"/>
-                    <opt data-xmlid="theme_bootswatch.theme_flatly_bs_variables,theme_bootswatch.theme_flatly" data-icon="/theme_bootswatch/static/src/img/flatly_thumbnail.png"/>
-                    <opt data-xmlid="theme_bootswatch.theme_journal_bs_variables,theme_bootswatch.theme_journal" data-icon="/theme_bootswatch/static/src/img/journal_thumbnail.png"/>
-                    <opt data-xmlid="theme_bootswatch.theme_simplex_bs_variables,theme_bootswatch.theme_simplex" data-icon="/theme_bootswatch/static/src/img/simplex_thumbnail.png"/>
-                    <opt data-xmlid="theme_bootswatch.theme_slate_bs_variables,theme_bootswatch.theme_slate" data-icon="/theme_bootswatch/static/src/img/slate_thumbnail.png"/>
-                    <opt data-xmlid="theme_bootswatch.theme_spacelab_bs_variables,theme_bootswatch.theme_spacelab" data-icon="/theme_bootswatch/static/src/img/spacelab_thumbnail.png"/>
-                    <opt data-xmlid="theme_bootswatch.theme_united_bs_variables,theme_bootswatch.theme_united" data-icon="/theme_bootswatch/static/src/img/united_thumbnail.png"/>
-                </content>
+    <template id="snippet_options" inherit_id="website.snippet_options">
+        <xpath expr="//div[@data-js='Theme']" position="replace">
+            <div data-js="Theme" data-selector="theme" data-no-check="true">
+                <we-button data-switch-theme="" data-no-preview="true">Switch Theme</we-button>
+                <we-select string="Bootswatch Theme">
+                    <we-button data-customize-website="" data-icon="/theme_bootswatch/static/src/img/bootswatch_default_thumbnail.png">Default</we-button>
+                    <we-button data-customize-website="theme_bootswatch.theme_cerulean_bs_variables,theme_bootswatch.theme_cerulean" data-icon="/theme_bootswatch/static/src/img/cerulean_thumbnail.png">Cerulean</we-button>
+                    <we-button data-customize-website="theme_bootswatch.theme_cosmo_bs_variables,theme_bootswatch.theme_cosmo" data-icon="/theme_bootswatch/static/src/img/cosmo_thumbnail.png">Cosmo</we-button>
+                    <we-button data-customize-website="theme_bootswatch.theme_cyborg_bs_variables,theme_bootswatch.theme_cyborg" data-icon="/theme_bootswatch/static/src/img/cyborg_thumbnail.png">Cyborg</we-button>
```

Comment:
Can't we use the same mechanism as `test_theme` to have a preview?
Right now if I am not mistaken the `data-icon` is useless, but a preview is really mandatory/nice to have.

Reply:
@sbuhl Maybe we should consider this when we add images and stuff in the left panel but as discussed with @rdeodoo I will not waste time with this theme since we plan to remove it... or did we change our minds about that ?